src/components/Main/NewSurveyPage/index.js

```diff
@@ -248,16 +272,40 @@ const NewSurveyPage = props => {
     />
   );
 
+  const pageToShow = surveyState.pages.length ? (
+    <div className={`newSurveyPage-newsurvey__survey-page`}>
+      {addedQuestions}
+      {newQuestion}
+    </div>
+  ) : null;
+
+  const handlePageClick = e => {
+    e.preventDefault();
+    const pageIndex = +e.target.innerText.split(' ')[1] - 1;
```

Comment:
Error-prone code, and also it is not clear what it does at first sight

Reply:
done